Add DevToolLab to the Tool Discovery section

Moksh45 opened the pull request.

moimikey commented:
thanks @Moksh45 for the submission! DevToolLab looks great, however, to fully accept this PR, please ensure that you acceptably categorize and link to individual tools that DevToolLab provide. Links should go to the tool for immediate usage.

Moksh45 commented:
Thanks for checking the MR, @moimikey.
I’ve removed the Tool Discovery section and added several tools, each with links to their respective pages on [DevToolLab](https://devtoollab.com/).
Please have a look at the updated MR. Thanks!

moimikey commented:
looks good! thank you!

moimikey commented:
dead link fail is from something else. merging.
